__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const fs = require("fs/promises")
+const endpoints = require("../endpoints.json")
+
```

Comment:
Before your tests, you can seed your test databases using the `db`, `seed`, and `data` you've imported in. If you go back to the notes or the code from the sprints last week, you'll find the code you need to do this!